pkg/builder/builder.go

```diff
@@ -140,7 +145,36 @@ func buildConfigVolumeMount(vdb *vapi.VerticaDB) corev1.VolumeMount {
 	}
 }
 
-// buildVolumeMounts returns the volume mounts to include in the sts pod spec
+// buildServerVolumeMounts returns the volume mounts to include
+// in the server container
+func buildServerVolumeMounts(vdb *vapi.VerticaDB) []corev1.VolumeMount {
+	volMnts := buildVolumeMounts(vdb)
```

Comment:
If we don't mount /tmp/scrutinize in the server container, then we will need to update scripts/capture-scrutinize.sh. We need to run scrutinize in the NMA container if that exists.

EDIT: probably no change here if the nma is the first container. That will default to the nma anyway.